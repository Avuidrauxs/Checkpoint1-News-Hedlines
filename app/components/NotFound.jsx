import React from 'react';


const NotFoundPage = () => (

  <div className="centered">
    <h1>404 - Page Not Found</h1>
    <p>No page of that sort exists</p>
  </div>
    );


export default NotFoundPage;
