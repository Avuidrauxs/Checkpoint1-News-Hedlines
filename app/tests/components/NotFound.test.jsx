import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import expects from 'expect';


import NotFound from '../../components/NotFound';

describe('NotFound', () => {
  it('should exist', () => {
    expects(NotFound).toExist();
  });
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NotFound />, div);
  });
  it('should render a snapshot', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
