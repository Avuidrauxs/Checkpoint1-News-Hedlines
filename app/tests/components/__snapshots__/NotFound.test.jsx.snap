// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render a snapshot 1`] = `
<div
  className="centered"
>
  <h1>
    404 - Page Not Found
  </h1>
  <p>
    No page of that sort exists
  </p>
</div>
`;
